rollup.config.js

```diff
@@ -27,7 +27,7 @@ function rollupConfig({ formats, prod }) {
             exclude: ['test/*'],
             typescript: require('typescript')
```

Comment:
The rollup plugin now requires `typescript` to be installed as a peer dependency. It is not needed to pass typescript anymore.

```suggestion
```